docs/man/docker-load.1.md

```diff
@@ -36,6 +36,9 @@ Restores both images and tags.
     fedora              heisenbug           58394af37342        7 weeks ago         385.5 MB
     fedora              latest              58394af37342        7 weeks ago         385.5 MB
 
+# See also
+**docker-save(1)** to save an image(s) to a tar archive.
```

Comment:
Should this mention STDOUT as well, as in the [CLI docs](https://docs.docker.com/reference/commandline/cli/#save)?